### DISCUSSION TITLE
求助：ssh 连接出错

### DISCUSSION THREAD

**Discussion by LiiiHua** [physics-data/faq#19] [Q&A] [answered]
git clone 时报错：
kex_exchange_identification: Connection closed by remote host
fatal: Could not read from remote repository.

网络连接正常，已经连接了sslvpn，之前提交和下载都正常。
.ssh/config 已修改过。

运行 ssh -vT git@github.com 显示：
lizehua@DESKTOP-ULPQBSC:~$ ssh -vT git@github.com
OpenSSH_8.2p1 Ubuntu-4ubuntu0.1, OpenSSL 1.1.1f  31 Mar 2020
debug1: Reading configuration data /home/lizehua/.ssh/config
debug1: /home/lizehua/.ssh/config line 1: Applying options for github.com
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: include /etc/ssh/ssh_config.d/*.conf matched no files
debug1: /etc/ssh/ssh_config line 21: Applying options for *
debug1: Connecting to 166.111.32.43 [166.111.32.43] port 47175.
debug1: Connection established.
debug1: identity file /home/lizehua/.ssh/id_rsa type 0
debug1: identity file /home/lizehua/.ssh/id_rsa-cert type -1
debug1: identity file /home/lizehua/.ssh/id_dsa type -1
debug1: identity file /home/lizehua/.ssh/id_dsa-cert type -1
debug1: identity file /home/lizehua/.ssh/id_ecdsa type -1
debug1: identity file /home/lizehua/.ssh/id_ecdsa-cert type -1
debug1: identity file /home/lizehua/.ssh/id_ecdsa_sk type -1
debug1: identity file /home/lizehua/.ssh/id_ecdsa_sk-cert type -1
debug1: identity file /home/lizehua/.ssh/id_ed25519 type -1
debug1: identity file /home/lizehua/.ssh/id_ed25519-cert type -1
debug1: identity file /home/lizehua/.ssh/id_ed25519_sk type -1
debug1: identity file /home/lizehua/.ssh/id_ed25519_sk-cert type -1
debug1: identity file /home/lizehua/.ssh/id_xmss type -1
debug1: identity file /home/lizehua/.ssh/id_xmss-cert type -1
debug1: Local version string SSH-2.0-OpenSSH_8.2p1 Ubuntu-4ubuntu0.1
kex_exchange_identification: Connection closed by remote host

**Comment by Harry-Chen** [comment]
是否配置使用了续老师提供的代理？

**Comment by LiiiHua** [comment]
已经配置代理，而且GitHub网站可以正常访问

**Comment by LiiiHua** [comment]
情况更新：
重启之后，连接sslvpn、配置代理并授权登录之后，仅能访问清华校内网站，其它所有网站均无法访问。
只连接sslvpn则可以正常上网。

**Comment by heroxbd** [comment, marked as answer]
查清了，是出口节点的包月账号欠费了

**Comment by LiiiHua** [comment]
> 
> 
> 查清了，是出口节点的包月账号欠费了

请问是我的问题吗？需要进行什么操作？

**Comment by heroxbd** [comment]
> > 查清了，是出口节点的包月账号欠费了
> 
> 请问是我的问题吗？需要进行什么操作？

不是，是我的问题。

**Comment by LiiiHua** [comment]
不过似乎没有其他同学遇到此问题。

**Comment by heroxbd** [comment]
> 不过似乎没有其他同学遇到此问题。

再试一下看看。我给账号充值了，我猜问题已经解决。

**Comment by LiiiHua** [comment]
一切正常了，谢谢您。

**Comment by heroxbd** [comment]
太好了！
